internal/controlplane/server.go

```diff
@@ -167,6 +170,7 @@ func (srv *Server) Run(ctx context.Context) error {
 
 // OnConfigChange updates the pomerium config options.
 func (srv *Server) OnConfigChange(cfg *config.Config) error {
+	srv.reproxy.Update(cfg)
```

Comment:
I seem to remember that order was important for this function. Is this correct?

Reply:
Sorry I don't know.